MRETestBed/Assets/TestBed Assets/Scripts/MREComponent.cs

```diff
@@ -88,9 +88,9 @@ void Start()
         if (!_apiInitialized)
         {
             MREAPI.InitializeAPI(
+                DefaultPrimMaterial,
                 behaviorFactory: new BehaviorFactory(),
                 textFactory: new MWTextFactory(SerifFont, SansSerifFont),
-                primitiveFactory: new MWPrimitiveFactory(DefaultPrimMaterial),
```

Comment:
>primitiveFactory [](start = 16, length = 16)

Did you intend to remove primitiveFactory?

Reply:
The initialization in the Testbed, yes. The constructor no longer takes arguments, so it's redundant with the default value.